src/server/README.md

```diff
@@ -1,4 +1,8 @@
-1. You Need to create Mongo db local/atlas. You can do this by navigating into ```/src/server/setup/``` and running  ```docker-compose up -d```.
+1. You Need to create Mongo db local/atlas. You can do this by navigating into ```/src/server/setup/```, then
```

Comment:
"You Need to create Mongo db local/atlas." 

This statement is a bit vague, but we are also working on documentation at this point.. so this can use some improvement.